src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -0,0 +1,259 @@
+              weight="regular"
+            >
+              Version: {version}
+            </Text>
+          </div>
+          <DownloadStandaloneLink
+            ariaLabel={`download ${name} version ${version} for ${prettyOSName}, architecture ${arch}`}
+            href={downloadsByOS[os][arch]}
+          />
+        </Card>
+      ))}
+    </>
+  )
+}
+
+const ChangelogSection = ({ selectedRelease }) => {
+  const { name, version } = selectedRelease
+  return (
+    <>
+      <Heading {...SHARED_HEADING_LEVEL_3_PROPS} slug="release-information">
+        Release information
+      </Heading>
+      <Card className={s.textAndLinkCard} elevation="base">
+        <div className={s.textAndLinkCardTextContainer}>
+          <Text className={s.textAndLinkCardLabel} size={200} weight="semibold">
+            Changelog
+          </Text>
+          <Text
+            className={s.textAndLinkCardVersionLabel}
+            size={200}
+            weight="regular"
+          >
+            Version: {version}
+          </Text>
+        </div>
+        <StandaloneLink
+          ariaLabel={`${name} version ${version} changelog`}
+          href={`https://github.com/hashicorp/waypoint/blob/v${version}/CHANGELOG.md`}
+          icon={<IconExternalLink16 />}
+          iconPosition="trailing"
+          openInNewTab
+          text="GitHub"
+          textSize={200}
+        />
+      </Card>
+    </>
+  )
+}
+
+const NotesSection = ({ selectedRelease }) => {
+  const { name, shasums, shasums_signature, version } = selectedRelease
+  return (
+    <>
+      <Heading
+        {...SHARED_HEADING_LEVEL_3_PROPS}
+        className={classNames(
+          SHARED_HEADING_LEVEL_3_PROPS.className,
+          s.specialSubHeading
+        )}
+        slug="notes"
+      >
+        Notes
+      </Heading>
+      <Text size={200}>
+        You can find the{' '}
+        <InlineLink
+          href={`https://releases.hashicorp.com/${name}/${version}/${shasums}`}
+          text={`SHA256 checksums for Waypoint ${version}`}
+          textSize={200}
+        />{' '}
+        online and you can{' '}
+        <InlineLink
+          href={`https://releases.hashicorp.com/${name}/${version}/${shasums_signature}`}
+          text="verify the checksums signature file"
+          textSize={200}
+        />{' '}
+        which has been signed using{' '}
+        <InlineLink
+          href="https://www.hashicorp.com/security"
+          text="HashiCorp's GPG key"
+          textSize={200}
+        />
+        .
+      </Text>
+    </>
+  )
+}
+
+const DownloadsSection = ({
+  latestVersionIsSelected,
+  packageManagers,
+  selectedRelease,
+}: DownloadsSectionProps): ReactElement => {
+  const downloadsByOS = useMemo(() => groupDownloadsByOS(selectedRelease), [
+    selectedRelease,
+  ])
+  const packageManagersByOS = useMemo(
+    () => groupPackageManagersByOS(packageManagers),
+    [packageManagers]
+  )
+
+  return (
+    <article>
+      <Card elevation="base">
```

Comment:
I wonder if it'd be worth it to allow `Card` to render `article` under the hood 💭

Reply:
Personally I think that's too heavy of an assumption for `Card` to make, especially since components like `CardLink` are built on top of it. If we made `Card` render an `article`, then `CardLink` would end up rendering an `article` element within an `a` element and I think we probably don't always want that as the default.

Reply:
Yeah, sorry I didn't mean by default, but with something like `asElement`

Reply:
Ah, I see! I think that could potentially be really useful!